From 5649a7365fcbabdbf26ff7549f733eb0da07d29c Mon Sep 17 00:00:00 2001
From: gouarin <loic.gouarin@gmail.com>
Date: Wed, 13 Dec 2017 16:12:28 +0100
Subject: [PATCH] remove asciidoc

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d2535b17..5fe04af4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -361,10 +361,10 @@ elseif (CMAKE_SYSTEM_NAME MATCHES "Darwin")
 endif ()
 
 set (CMAKE_PYTHON_VERSION 2.7 2.6 2.5 2.4)
-find_package (PythonInterp)
-find_package (AsciiDoc)
+# find_package (PythonInterp)
+# find_package (AsciiDoc)
 
-cmake_dependent_option (WITH_DOC "Build Reference Guide documentation (requires DocBook)" ON
+cmake_dependent_option (WITH_DOC "Build Reference Guide documentation (requires DocBook)" OFF
                        "PYTHONINTERP_FOUND;ASCIIDOC_FOUND" OFF)
 
 if (MSVC)
-- 
2.14.1

