diff --git a/CMakeLists.txt b/CMakeLists.txt
index d2535b1..ebeb2fd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -857,7 +857,7 @@ if (BUILD_SHARED)
   endif ()
 
   if (RT_LIBRARY)
-    target_link_libraries (libzmq ${RT_LIBRARY})
+    target_link_libraries (libzmq -lrt)
   endif ()
 endif ()
 
@@ -881,7 +881,7 @@ if (BUILD_SHARED)
       target_link_libraries (${perf-tool} libzmq ${OPTIONAL_LIBRARIES})
 
       if (RT_LIBRARY)
-        target_link_libraries (${perf-tool} ${RT_LIBRARY})
+        target_link_libraries (${perf-tool} -lrt)
       endif ()
 
       if (ZMQ_BUILD_FRAMEWORK)
